SPECS-EXTENDED/lensfun/lensfun.spec

```diff
@@ -0,0 +1,383 @@
+%if !0%{?bootstrap} && (0%{?fedora} || 0%{?rhel} > 6)
```

Comment:
Could you redo this by removing the references to fedora/rhel, here and line#74, thanks